src/cli/grpc_cli_server.cpp

```diff
@@ -48,6 +48,24 @@ class PosCliServiceImpl final : public PosCli::Service {
   {
     grpc::Status status = pc->ExecuteSetSystemPropertyCommand(request, reply);
     
+    return status;
+  }
+
+    grpc::Status
```

Comment:
Minor : Space

Reply:
Thank you for the comment. I've just fixed it.